stable/rabbitmq/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if and .Values.metrics.enabled .Values.metrics.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "rabbitmq.fullname" . }}
+  {{- if .Values.metrics.serviceMonitor.namespace }}
+  namespace: {{ .Values.metrics.serviceMonitor.namespace }}
+  {{- end }}
+  labels:
+    app: {{ template "rabbitmq.name" . }}
+    chart: {{ template "rabbitmq.chart" .  }}
+    heritage: "{{ .Release.Service }}"
+    release: "{{ .Release.Name }}"
+    {{- if .Values.metrics.serviceMonitor.additionalLabels }}
+{{ toYaml .Values.metrics.serviceMonitor.additionalLabels | indent 4 }}
+    {{- end }}
+spec:
+  endpoints:
+  - port: metrics
+    interval: 30s
```

Comment:
The interval should be configurable. See PostgreSQL for instance:
https://github.com/helm/charts/blob/master/stable/postgresql/templates/servicemonitor.yaml#L21

Reply:
The `scrapeTimeout` should be also configurable. See PostgreSQL for instance:
https://github.com/helm/charts/blob/master/stable/postgresql/templates/servicemonitor.yaml#L24

Reply:
Done.